### DISCUSSION TITLE
Add an icon for discontinued apps and services

### DISCUSSION THREAD

**Discussion by InterstellarOne** [pluja/awesome-privacy#410] [Ideas]
There's a lot of potentially discontinued apps in this list, and many of these still work, so I was thinking instead of removing them, or keeping them and not giving any notice it would be useful to just add an icon for discontinued apps (and also maybe a legend for the icons)

**Comment by pluja** [comment]
That's a good idea! I will be adding the icon to the apps I know are discontinued.

If you or anyone knows any other ones, please contribute with a PR or open an issue with the list of discontinued apps.

**Reply by InterstellarOne** [reply]
Okay! let me know when you choose an icon so I can add it to apps that I know.

**Comment by mfathurr** [comment]
If I may add, if possible it would be better if after the name all of the alternative application there are icons of any device that can be used in the alternative application.

This is to make it easier for people who don't follow/understand technology (newbie/beginner/other names) and people who are just starting to switch little by little to FOSS or AOSP.


Example:

Obtainium [icon android] [icon ios] [and other icons] - Description of the application.

Bitwarden [icon android] [icon ios] [icon windows] [icon linux] [icon mac os] [icon web] [and other icons] - Description of the application.
